src/redux-query-sync.js

```diff
@@ -71,7 +71,7 @@ function ReduxQuerySync({
         // For each parameter value that changed, call the corresponding action.
         Object.keys(queryValues).forEach(param => {
             const value = queryValues[param]
-            if (value !== lastQueryValues[param]) {
+            if (value && value !== lastQueryValues[param]) {
```

Comment:
This does not seem right: if the value is falsy, the action would not be invoked. I think you can remove this check if you move your other change one line down.